Dockerfile

```diff
@@ -18,25 +18,14 @@ ARG GOTESTSUM_VERSION=v1.9.0
 ARG DELVE_VERSION=v1.21.0
 
 ARG GO_VERSION=1.21
-ARG ALPINE_VERSION=3.19
+ARG ALPINE_VERSION=3.20
 ARG XX_VERSION=1.4.0
```

Comment:
This will still use `alpine:edge` for riscv64 cross-comp with this version of xx: https://github.com/tonistiigi/xx/blob/81db39e375537bb5711ecfdcb1526e4c9ea4c0e1/src/xx-apk#L49-L53

Opened https://github.com/tonistiigi/xx/pull/157/commits/33ae6f067f743833e1a7e32ecde84d0f31767fb1 as follow-up.